build-system/tasks/bundle-size.js

```diff
@@ -350,6 +351,14 @@ async function performBundleSizeCheck() {
   if (argv.on_skipped_build) {
     return await skipBundleSize();
   } else {
+    if (argv.on_pre_push_build) {
+      const files = gitDiffNameOnlyMaster();
+      const runtimeFiles = files.filter(fileName => fileName.startsWith('src'));
```

Comment:
This is a pretty crude check to see if a diff-ed file is in runtime. Let me know if there's a better one.